core/src/trezor/lvglui/i18n/locales/ja.py

```
@@ -1025,8 +1024,29 @@
     "タッチ",
+    "ホーム画面",
     "変更",
-    "壁紙が追加されました",
-    "今すぐ壁紙として設定しますか？",
+    "ロック画面に設定",
+    "ホーム画面に設定",
+    "プレビュー",
+    "OneKey Appから壁紙を追加",
+    "My OneKeyで画像をアップロード > お使いのOneKeyデバイスを選択 > 壁紙を追加します。",
+    "コレクション",
+    "暗号資産ネットワークが見つかりませんか？USB または Bluetooth で接続してみてください",
```

Ruff (0.14.3)

String contains ambiguous `？` (FULLWIDTH QUESTION MARK). Did you mean `?` (QUESTION MARK)?

(RUF001)